### ISSUE TITLE
Missing documentation on compiling model

### ISSUE THREAD

**Issue by yardstick17** [farrell236/ResNetAE#2]
What would you recommend to compile the model?
- What optimizer, and  loss to choose

`model.compile(loss="binary_crossentropy", optimizer=optimizers.Adam(lr=1e-3))`
